src/meta/src/barrier/recovery.rs

```diff
@@ -663,9 +663,17 @@ impl GlobalBarrierManager {
                     request_id: Uuid::new_v4().to_string(),
                 })
                 .await
+                .map(|resp| (worker_node.id, resp))
         });
 
-        try_join_all(futures).await?;
+        let results = try_join_all(futures).await?;
+        for (worker_id, result) in results {
+            tracing::info!(
+                "cause of previous actor failure in worker node {}: {}",
+                worker_id,
+                result.previous_actor_failure_cause
+            );
+        }
```

Comment:
Let's use structured logging here

Reply:
QQ: you mean represent worker_id as structured field, right? But it seems to be no global naming rules for structured field. I can use `worker=worker_id`, or `worker_id=worker_id`, etc.

Reply:
```suggestion
            tracing::info!(
                worker_id
                reason=result.previous_actor_failure_cause
                "previous actor failed in worker node",
            );
        }
```

Reply:
Yeah no global naming rules. Structured logs are easier to process, e.g. when they're output as json.

Reply:
Seems not resolved yet? 👀

Reply:
Seems not resolved yet? 👀